code/controllers/subsystem/pai.dm

```diff
@@ -195,4 +219,6 @@ SUBSYSTEM_DEF(pai)
 	var/description
 	var/role
 	var/comments
+	var/secondary_color = "#DA1B1B"
+	var/color = "#2DDF1D"
```

Comment:
Considering my earlier comment about not needing a secondary color since we never really modify it manually.
```suggestion
	var/color = "#2DDF1D"
```